### PR TITLE
Release Scala 2.13 artifacts

### PR DESCRIPTION
**Pull request by cocreature**
I believe this is somewhat reasonable but as usual for these type of changes, I have no way of testing this and it’s a bunch of bash and yaml so it probably doesn’t work.

The most annoying part is the check if we can release 2.13 artifacts. I don’t see a way around that unfortunately.

changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ci/build.yml [hunk 1]
```diff
@@ -70,6 +70,47 @@ jobs:
         trigger_sha: '$(trigger_sha)'
     - template: report-end.yml
 
+- job: Linux_scala_2_13
```
Comment (cocreature) [inline ci/build.yml:73]: this is bad, this is going to fail for older releases. I’m not quite sure how to fix that unfortunately without starting to parse semvers in bash.

File: ci/build-unix.yml [hunk 2]
```diff
@@ -5,6 +5,7 @@ parameters:
   is_release: ''
   name: ''
   release_tag: ''
+  scala_version: '2.12.12'
```
Comment (SamirTalwar-DA) [inline ci/build-unix.yml:8]: I'd love to have this hard-coded in fewer places. Can we set it to the empty string by default? We might need to handle this in _scala_version.bzl_.

Comment (cocreature) [reply ci/build-unix.yml]: Very valid point 9ea71dda2e5e7e59becd037149b96736f2f4bf9b

**Review by stefanobaghino-da — APPROVED** [review]

File: ci/build-unix.yml (hunk 2 above)
Comment (SamirTalwar-DA) [reply ci/build-unix.yml]: Yay!

**Review by SamirTalwar-DA — APPROVED** [review]
I haven't reviewed the YAML thoroughly because, well, I value my sanity. But I tried.

**Pull request merged**